frontend/test/playwright/utils/expect-snapshot.ts

```diff
@@ -96,7 +137,7 @@ export const expectSnapshot: ExpectSnapshot = async (
   await turnOnDarkMode(page, dir ?? "ltr")
 
   // Wait for the theme to change.
-  await sleep(200)
+  await sleep(100)
```

Comment:
This was the original value. I thought we needed to increase it to fix the modal rendering, but turns out that switching the theme was done incorrectly before. When the modal is open, the theme switcher is covered by it. We need to close the modal, switch the theme, and then re-open the modal before taking the snapshot.